commands/test/test-command.ts

```diff
@@ -156,19 +156,67 @@ export const sendQuestion = async (interaction: any) => {
-            interaction.user.send("Als nächstes schreibst du deinem Partner, indem du auf seinen Namen auf dem Contraversum-Server klickst 👆 und ihm eine Nachricht sendest.");
-            interaction.user.send("Dies sind drei Fragen bei denen ihr euch unterscheidet:");
+            const guildId = process.env.GUILD_ID;
+            if (!guildId) {
+                console.error('GUILD_ID is not defined in .env');
+                return;
+            }
+            const guild: Guild | undefined = client.guilds.cache.get(guildId);
+            if (!guild) {
+                console.error('Guild not found');
+                return;
+            }
 
-            // Send the best match that they have been matched with the user
-            const bestMatchUser = await client.users.fetch(bestMatch.userId);
-            if (bestMatchUser) {
-                bestMatchUser.send(`Hey 👋, du wurdest mit: **@${interaction.user.username}** gematched.`);
-                bestMatchUser.send("Ihr habt euch bei diesen drei Fragen beispielsweise unterschieden:");
+            const member = guild.members.cache.get(interaction.user.id);
+            if (!member) {
+                console.error('Member not found');
+                return;
+            }
+            const bestId = bestMatch.userId
+            console.log(bestId);
+
+            const bestMember = await guild.members.fetch(bestId);
+            if (!bestMember) {
+                console.error('bestMember not found');
+                return;
             }
-            conversationStarter(interaction, bestMatch.userVector, userResponses, bestMatchUser)
+
+            const matchesCategory = guild.channels.cache.find((category: any) => category.name === 'matches' && category.type === 4);
+
+            const channelName = `match-${interaction.user.username}-${bestMatch.username}`;
+            console.log("current channel name: " + channelName)
+
+            const textChannel = await guild.channels.create({
+                parent: matchesCategory?.id,
+                name: channelName.toLowerCase(),
+                type: 0,
+            });
+
+            await textChannel.permissionOverwrites.edit(member, {
+                ViewChannel: true,
+                SendMessages: true,
+            });
+            await textChannel.permissionOverwrites.edit(bestMember, {
+                ViewChannel: true,
+                SendMessages: true,
+            });
+            const everyone = await guild.roles.everyone;
+
+            await textChannel.permissionOverwrites.edit(everyone, {
+                ViewChannel: false,
+            });
+
+            await textChannel.send(`Hallo ${member} 👋, hallo ${bestMember} 👋, basierend auf unserem Algorithmus wurdet ihr als Gesprächspartner ausgewählt. Bitte vergesst nicht respektvoll zu bleiben. Viel Spaß bei eurem Match!`);
+            await textChannel.send(`Bei diesen drei Fragen seid ihr nicht einer Meinung: `);
```

Comment:
Maybe change it to: 'Bei bspw. diesen drei Fragen seid ihr nicht einer Meinung: '